tasks/Resources/Provisioning/Hive/OSP/hive_osp_cluster_template.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: hive.openshift.io/v1
+  kind: ClusterPool
+  metadata:
+    name: ${infrastructure_configurations}[hive_cluster_name]-pool
+    namespace: ${infrastructure_configurations}[hive_claim_ns]
+  spec:
+    baseDomain: ${infrastructure_configurations}[aws_domain]
+    imageSetRef:
+      name: rhods-imageset-ocp-${infrastructure_configurations}[ocp_version] # see line 99
+    inventory:
+      - name: ${infrastructure_configurations}[hive_cluster_name]-conf # see line 71
+    platform:
+      openstack:
+        cloud: ${infrastructure_configurations}[osp_cloud_name]
+        credentialsSecretRef:
+          name: ${infrastructure_configurations}[hive_cluster_name]-cred # see line 90
+    installConfigSecretTemplateRef:
+      name: ${infrastructure_configurations}[hive_cluster_name]-sec # see line 30
+    size: 1
+    maxSize: 1
+    maxConcurrent: 1
+    runningCount: 1
+    skipMachinePools: true
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: ${infrastructure_configurations}[hive_cluster_name]-sec
+    namespace: ${infrastructure_configurations}[hive_claim_ns]
+  type: Opaque
+  stringData:
+    install-config.yaml: |
+      apiVersion: v1
+      baseDomain: ${infrastructure_configurations}[aws_domain]
+      compute:
+      - name: worker
+        platform:
+          openstack:
+            type: ${infrastructure_configurations}[osp_flavor]
+        replicas: 3
+      controlPlane:
+        name: master
+        platform:
+          openstack:
+            type: ${infrastructure_configurations}[osp_flavor]
+        replicas: 3
+      metadata:
+        name: ${infrastructure_configurations}[hive_cluster_name]-install
+      networking:
+        clusterNetwork:
+        - cidr: 10.128.0.0/14
+          hostPrefix: 23
+        machineNetwork:
+        - cidr: 192.169.0.0/16
```

Comment:
should these value be stored into a variable too?

Reply:
On other provisioning we do not externalize the CIDRs, since the benefit of it is just for use cases of Networking tests. 
We can add a new PR to externalize these CIDRs to be passed by configuration, and Jenkins later.